Manual.md

```diff
@@ -461,7 +462,11 @@ the generated `binary packages` have been modified.
 - `short_desc` A string with a brief description for this package. Max 72 chars.
 
 - `version` A string with the package version. Must not contain dashes or underscore
-and at least one digit is required. Shell's variable substition usage is not allowed.
+and at least one digit is required. Using bash's pattern substitution, prefix and suffix
+matching isn't supported, since this field needs to be parsed by `xbps-checkvers(1)`.
```

Comment:
I'm for not using variables in version and constructing other variables from version, as xupdate from xtools' issue works better then.

Reply:
I can't find `xupdate`...

This would mean that the following packages are "incorrect", and we'd have to agree that they are better off with raw `version` fields:

```
srcpkgs/EmptyEpsilon/template:version="${_ver_major}.${_ver_minor}.${_ver_patch}"
srcpkgs/ImageMagick/template:version="${_majorver}.${_patchver}"
srcpkgs/ImageMagick6/template:version="${_majorver}.${_patchver}"
srcpkgs/cross-x86_64-w64-mingw32/template:version=$_mingw_version
srcpkgs/dialog/template:version="${_distver}.${_date}"
srcpkgs/dmraid/template:version="${_distver}.${_patchver}"
srcpkgs/ftgl/template:version="${_distver}${_distrcver}"
srcpkgs/gcc-multilib/template:version=${_majorver}.1pre1
srcpkgs/gcc/template:version=${_patchver}pre1
srcpkgs/gcc6/template:version=${_minorver}.0
srcpkgs/google-chrome/template:version="${_chromeVersion}.${_chromeRevision}"
srcpkgs/hfsprogs/template:version="${_distver}.linux${_patchver}"
srcpkgs/j/template:version=${_vmaj}.${_vmin}
srcpkgs/libax25/template:version="${_distver}${_patchver}"
srcpkgs/libedit/template:version="${_datever}.${_distver}"
srcpkgs/libpurple-facebook/template:version=${_version}.${_rev}
srcpkgs/libreadline8/template:version="${_dist_ver}.${_patch_ver}"
srcpkgs/libtcd/template:version=${_distver}.${_distrev}
srcpkgs/lynx/template:version="${_distver}.${_patchver}"
srcpkgs/man-pages-posix/template:version="${_distver}${_revver}"
srcpkgs/mcwm/template:version="${_distver}.${_patchver}"
srcpkgs/openjdk10-bootstrap/template:version="10.0.2p${_jdk_build}"
srcpkgs/openjdk11/template:version="${_java_ver}.${_java_min_ver}.${_java_sec_ver}+${_jdk_update}"
srcpkgs/openjdk7-bootstrap/template:version="${_java_ver}u${_jdk_update}b${_jdk_build}"
srcpkgs/openjdk8/template:version="${_java_ver}u${_jdk_update}b${_jdk_build}"
srcpkgs/openjdk9-bootstrap/template:version="9.0.4p${_jdk_build}"
srcpkgs/pcc/template:version=${_distver}.${_snapshot}
srcpkgs/python3-apsw/template:version=${_distver}${_patchver}
srcpkgs/simutron/template:version=1.0.1.${_rev}
srcpkgs/tcc/template:version=${_distver}.${_voidver}
srcpkgs/x264/template:version="${_snap_date}.${_snap_rev}"
```